R/render.r

```diff
@@ -52,12 +52,19 @@ render_page <- function(pkg = ".", name, data, path = "", depth = NULL, quiet =
   # Potential opt-out of syntax highlighting CSS
   data$needs_highlight_css <- !isFALSE(pkg$meta[["template"]]$params$highlightcss)
 
+  # Search index location
+  data$`search-index` <- paste0("/", pkg$prefix, "search.json")
+
   # render template components
   pieces <- c(
     "head", "navbar", "header", "content", "docsearch", "footer",
     "in-header", "after-head", "before-body", "after-body"
   )
 
+  if (pkg$bs_version > 3) {
+    pieces <- pieces[pieces != "docsearch"]
```

Comment:
So we turn it on by default?

Reply:
Good point, at the moment to turn it off you'd need to use a custom navbar template. Should there be a configuration switch?

Reply:
I don't think so; if it's useful we might as well turn it on by default.